frontend-maven-plugin assumes target platform is x64?!

maven fails to install nodejs-v4.4.4 for ppc64le architecture with the appended  error message:
repository location : http://nodejs.org/dist/v4.4.4/
changing   <nodeVersion>v4.4.4</nodeVersion> to  <nodeVersion>v4.4.4-linux-ppc64le</nodeVersion> does not work either. Please help.
 ------------------------------------------------------------------------
[INFO] Building Zeppelin: web Application 0.5.5-incubating
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.6.1:clean (default-clean) @ zeppelin-web ---
[INFO] 
[INFO] --- maven-checkstyle-plugin:2.13:check (checkstyle-fail-build) @ zeppelin-web ---
[INFO] 
[INFO] 
[INFO] --- maven-resources-plugin:2.7:copy-resources (copy-resources) @ zeppelin-web ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 17 resources
[INFO] 
[INFO] --- maven-enforcer-plugin:1.3.1:enforce (enforce) @ zeppelin-web ---
[INFO] 
[INFO] --- frontend-maven-plugin:1.0:install-node-and-npm (install node and npm) @ zeppelin-web ---
[INFO] Installing node version v4.4.4
[INFO] Unpacking /home/sanjar/.m2/repository/com/github/eirslett/node/4.4.4/node-4.4.4-linux-x64.tar.gz into /home/sanjar/development/zeppelin-0.5.5-incubating/zeppelin-web/node/tmp
[INFO] Copying node binary from /home/sanjar/development/zeppelin-0.5.5-incubating/zeppelin-web/node/tmp/node-v4.4.4-linux-x64/bin/node to /home/sanjar/development/zeppelin-0.5.5-incubating/zeppelin-web/node/node
[INFO] Installed node locally.
[INFO] Installing npm version 2.15.1
[INFO] Downloading http://nodejs.org/dist/npm-2.15.1.tgz to /home/sanjar/.m2/repository/com/github/eirslett/npm/2.15.1/npm-2.15.1.tar.gz
[INFO] No proxies configured
[INFO] No proxy was configured, downloading directly
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 6.610 s
[INFO] Finished at: 2016-05-14T06:11:52+00:00
[INFO] Final Memory: 23M/316M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.0:install-node-and-npm (install node and npm) on project zeppelin-web: Could not download npm: Got error code 404 from the server. -> [Help 1]



pom.xml
          <execution>
            <id>install node and npm</id>
            <goals>
              <goal>install-node-and-npm</goal>
            </goals>
            <configuration>
         <!--      <nodeVersion>v0.10.22</nodeVersion>
              <npmVersion>1.3.8</npmVersion> -->
              <nodeVersion>v4.4.4</nodeVersion>
              <npmVersion>2.15.1</npmVersion>
              <downloadRoot>http://nodejs.org/dist/</downloadRoot>
            </configuration>
          </execution>

--- a/frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/Platform.java
+++ b/frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/Platform.java
@@ -1,8 +1,13 @@
 package com.github.eirslett.maven.plugins.frontend.lib;
 
-enum Architecture { x86, x64;
+enum Architecture { x86, x64, ppc64le;
     public static Architecture guess(){
-        return System.getProperty("os.arch").contains("64") ? x64 : x86;
+        String arch = System.getProperty("os.arch");
+        if (arch.equals("ppc64le")) {
+            return ppc64le;
+        } else {
+            return arch.contains("64") ? x64 : x86;
+        }
     }
 }
 

